jenkins/test-history/buckets.json

```diff
@@ -0,0 +1,11 @@
+{
+    "gs://kubernetes-jenkins/logs/": {
+        "prefix": ""
+    },
+    "gs://kube_azure_log": {
+        "prefix": "azure$"
+    },
+    "gs://rktnetes-jenkins/logs/": {
+        "prefix": "rktnetes$"
```

Comment:
They were used to parse job names back into buckets (azure$) before PR #20 did away with it. It could be `azure-` after that is merged.